cost/azure/rightsize_netapp_files/azure_rightsize_netapp_files.pt

```diff
@@ -0,0 +1,1385 @@
+name "Azure Rightsize NetApp Files"
+rs_pt_ver 20180301
+type "policy"
+short_description "Cheks for oversized NetApp Capacity Pools and suggest recommendations to save money. See the [README](https://github.com/flexera-public/policy_templates/tree/master/cost/azure/rightsize_netapp_files/) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
```

Comment:
Perhaps "Checks for oversized NetApp Capacity Pools and Volumes. See ..."

Reply:
Phew! A typo, thanks!